packages/library/src/commonMain/kotlin/io/realm/internal/RealmObjectCompanion.kt

```diff
@@ -23,6 +23,7 @@ import kotlin.reflect.KMutableProperty1
 @Suppress("VariableNaming")
 interface RealmObjectCompanion {
     val `$realm$fields`: List<KMutableProperty1<*, *>>?
+    val `$realm$indexedProps`: List<KMutableProperty1<*, *>>?
```

Comment:
We should probably use the same naming for properties? The above is called `$realm$fields` so this should probably be `$realm$indexedFields`.

Or we rename `$realm$fields` to `$realm$properties`...either seems fine to me.

Reply:
turns out it is even needed, so will remove.